README.md

```diff
@@ -124,8 +124,7 @@ Perl is used for the configure script, running the automated tests and examples
 included in this source tree, and generating Makefiles or Visual Studio project
 files.
 
-On Windows we recommend the use of [ActiveState Perl](
-https://www.activestate.com/activeperl).
+On Windows we recommend the use of [Straweberry Perl](https://strawberryperl.com).
```

Comment:
If we're changing this, [INSTALL.md](https://github.com/objectcomputing/OpenDDS/blob/master/INSTALL.md#compiling) and [configure.cmd](https://github.com/objectcomputing/OpenDDS/blob/4f3eba4be2a7f4d7412d4d041852bb2cbd79719a/configure.cmd#L8) also reference ActiveState Perl.

Reply:
Our builds use activestate without problems

Reply:
It's still possible to use it, but we've had a few minor issues with it. I think our main problems with it are that [it doesn't come with `Win32::Process`](https://stackoverflow.com/questions/64320132/what-is-win32-process-pm-in-inc/64321097) included and we couldn't use it build OpenDDS for Android on Windows, so [the instructions for that already say to use Strawberry Perl for that](https://github.com/objectcomputing/OpenDDS/blob/master/docs/android.md#building-on-windows).

Reply:
Ok, maybe mention this somewhere in the docu that ActiveState could be problematic?